ui/src/Credentials.ts

```diff
@@ -28,11 +28,11 @@ function isCredentials(credentials: any): credentials is Credentials {
 const CREDENTIALS_STORAGE_KEY = 'credentials';
 
 export function storeCredentials(credentials?: Credentials): void {
-  localStorage.setItem(CREDENTIALS_STORAGE_KEY, JSON.stringify(credentials));
+  sessionStorage.setItem(CREDENTIALS_STORAGE_KEY, JSON.stringify(credentials));
```

Comment:
The `localStorage` was causing issues demoing where if a tab was left open, it would no longer be refreshable since the credentials were overwritten by the new tab. I think it would make sense to have the DamlHub login use `localStorage`, but for the parties dropdown which is mainly for demoing, use `sessionStorage`.

Reply:
Does `sessionStorage` help here? Data is still stored per origin (domain), so I would expect that having multiple tabs open would still overwrite the selected credentials in storage

Reply:
Yup, it seems to be per individual tab (and I can confirm it does work) -

From https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage:
"Opening multiple tabs/windows with the same URL creates sessionStorage for each tab/window."

Reply:
Nice!